infra/storybook/deploy.sh

```
@@ -0,0 +1,25 @@
+#
+# Secrets resolve from 1Password via `op read`, so be signed in first
+# (`op signin`). Only 1Password *references* live in this file — never the
+# resolved secret values. Defaults to a read-only `preview`; pass `up` to apply:
+#
+#   ./deploy.sh            # pulumi preview (no changes)
+#   ./deploy.sh up         # pulumi up (creates/updates real resources)
+#
+set -euo pipefail
+cd "$(dirname "$0")"
+
+export VERCEL_API_TOKEN="$(op read 'op://BEEP_SECRETS/BEEP_SECRETS/CLOUD_VERCEL_API_TOKEN')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)